### PR TITLE
Draft proofread: minor repo cleanup

### PR DESCRIPTION
**Pull request by mathieuboudreau**
Here's a link to the MyBinder running on my repo: https://mybinder.org/v2/gh/mathieuboudreau/binder-tuto/master?filepath=notebooks%2Fnilearn-example.ipynb

Changes:

* Made the Binder button point directly to the notebook instead of the directory tree.
* Add two missing requirements to be able to run on MyBinder (scipy and sklearn).
  * (I imagine that you first wrote/tested this on your local, which might of had those reqs already available)
* The figure wasn't loading for me; I had to add `%matplotlib inline` in the notebook for it to work.
* Cleaned up README.md with some suggested changes.

**NOTE**
One more change that I think is warranted, but did not do here, is the section talking about the ipynb checkpoints. I don't think that's what they are used for; notebooks outputs (but not variable states) are saved in the notebook regardless of if the checkpoints are included in the repo I believe. I think checkpoints are more for a given session, where you can revert to older versions of your notebook.

Also, I don't know if we really "upload" to MyBinder, we simply give it the repo name and it builds an image that gets saved, but that's really a technicality.

### CODE REVIEW

**Comment by ltetrel** [comment]
You are right, I just checked and indeed the checkpoints are not mandatory.
I will update in consequence the README !

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 
 ### What is Binder ?
-[Binder](https://github.com/jupyterhub/binderhub) allows you to open notebooks from a git repo in an executable environment, accessible by anyone in the web. This make your work reproducible and shareable verry easy like never before. 
+[Binder](https://github.com/jupyterhub/binderhub) allows you to open notebooks saved in a GitHub repo in an executable environment, accessible by anyone using a web browser. This make your work reproducible and shareable very easy like never before. 
 
 It ties together many technologies :
 * [Docker](https://www.docker.com/), a tool that emphasizes reproducibility by packaging your applications into containers to run them in any environment.
-* [Jupyter hub](https://jupyterhub.readthedocs.io/en/latest/), that provides the infrastructure to share multiple instances of notebooks among many user.
+* [repo2docker](https://github.com/jupyter/repo2docker), a tool that converts GitHub repositories into Jupyter-enabled Docker images.
+* [JupyterHub](https://jupyterhub.readthedocs.io/en/latest/), which provides the infrastructure to share multiple instances of notebooks among many user.
```
Comment (emdupre) [inline README.md:12]:
```suggestion
* [JupyterHub](https://jupyterhub.readthedocs.io/en/latest/), which provides the infrastructure to share multiple instances of notebooks among many users.
```

File: README.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
 
 ### How to upload a work on Binder ?
 
-1. Have a github account (GitLab, Gist also supported).
+1. Have a GitHub account (GitLab, Gist also supported).
 
-2. Create a repository with at leat one notebook (your work), and its dependencies in a file `requirements.txt`.
+2. Create a repository with at leat one notebook (your work), and its dependencies listed in a file `requirements.txt`.
```
Comment (emdupre) [inline README.md:18]: They can also have a Dockerfile directly -- is that worth noting ?

File: notebooks/nilearn-example.ipynb [hunk 3]
```diff
@@ -40,14 +42,16 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/ltetrel/.local/lib/python3.6/site-packages/scipy/ndimage/measurements.py:272: DeprecationWarning: In future, it will be an error for 'np.bool_' scalars to be interpreted as an index\n",
+      "/srv/conda/lib/python3.7/site-packages/nilearn/plotting/displays.py:600: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working\n",
```
Comment (emdupre) [inline notebooks/nilearn-example.ipynb:45]: Do you want to catch these warnings for the purposes of the demonstration ? The code to do so could go in the first block and look like:

```python
import warnings
warnings.filterwarnings('ignore')
```

**Review by emdupre — COMMENTED** [review]
Adding in a few suggestions !

**Comment by emdupre** [comment]
Oops, I reviewed too late ! Sorry !!

@ltetrel pinging just so you can see I had a few additional comments if you want to incorporate them !

File: notebooks/nilearn-example.ipynb (hunk 3 above)
Comment (ltetrel) [reply notebooks/nilearn-example.ipynb:45]: nice I did not know about this !

**Pull request merged**